src/mutable_data.rs

```diff
@@ -29,9 +29,9 @@ pub struct SeqMutableData {
     permissions: BTreeMap<PublicKey, PermissionSet>,
     /// Version should be increased for any changes to MutableData fields except for data.
     version: u64,
-    /// Contains a set of owners of this data. DataManagers enforce that a mutation request is
-    /// coming from the MaidManager Authority of the Owner.
-    owners: PublicKey,
+    /// Contains the owner of this data. DataManagers enforce that a mutation request is coming from
```

Comment:
Just noticed that this comment mentions legacy names from Vaults.
Could you please change it to something like
```rust
/// Contains the public key of an owner or owners of this data.
/// Data Handlers in vaults enforce that a mutation request has a valid signature of the owner.
```
?